browser/css/jsdialogs.css

```diff
@@ -1196,6 +1196,7 @@ input[type='number']:hover::-webkit-outer-spin-button {
 .formulabar.unotoolbutton {
 	margin-inline-end: 4px !important;
 	opacity: 0.8;
+	width: 37px;
```

Comment:
Thanks and thanks for explaining to me where the jump occurs:


https://user-images.githubusercontent.com/65948705/183068080-7237ca42-e01d-4532-a55f-8c957316e272.mov



This fixes it! But having that extra space always after the `#FunctionDialog` looks a bit odd hm... maybe it would be better to refrain from changing every unotoolbutton in the formulabar and instead make sure `#acceptformula` occupies the exact same space as the former icon AutoSumMenu (that occupies more right now due to the additional arrow):

```css
#acceptformula {
    margin-inline-start: 3px;
```

What do you think?

Reply:
Makes sense, pushed as you suggested :)